src/retry-queue.js

```diff
@@ -6,41 +6,29 @@ export class RetryQueue extends RequestQueueScaffold {
         super()
         this.captureMetrics = captureMetrics
         this.isPolling = false
-        this._requestRetriesMap = {} // <RequestId, number>
-        this._counterToQueueMap = {}
-        this._pollerCounter = 1
-        this._areWeOnline = 'onLine' in window.navigator ? window.navigator.onLine : true
-        this._offlineBacklog = []
+        this.queue = []
+        this.areWeOnline = true
 
         if ('onLine' in window.navigator) {
+            this.areWeOnline = window.navigator.onLine
```

Comment:
not all browsers support this, although I guess having it as `undefined` achieves the same purpose as `false`, despite a bit messier types

Reply:
thanks for taking over and feel free to change anything... I never really ran it :D